omero/training-server/playbook.yml

```diff
@@ -217,6 +251,14 @@
     omero_server_system_managedrepo_group: managed_repo_group
     omero_server_datadir_managedrepo_mode: u=rwX,g=srwX,o=rX,+t
     omero_server_datadir_chown: True
+    omero_server_release_vars: "{{ omero_server_release_override | default('5.4.6') }}"
```

Comment:
The `_vars` suffix on some of these variable names isn't needed. Could you remove it?